.github/workflows/post-merge.yml

```diff
@@ -0,0 +1,32 @@
+name: Post Merge
+
+on:
+  push:
+    branches:
+      - master
+
+permissions:
+  contents: read
+
+concurrency: nginx-proxy-post-merge
+
+jobs:
+  image-tests:
+    name: Run Docker Image Tests
+    uses: ./.github/workflows/run-tests.yml
+
+  unit-tests:
```

Comment:
I think you don't want this task right? There aren't any npm tests to run in here right?

Reply:
Done!